devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -6,7 +6,7 @@ did-auth-proxy-helm:
   replicaCount: 1
 
   image:
-    repository: 098061033856.dkr.ecr.us-west-2.amazonaws.com/did-auth-proxy
+    repository: 984870885661.dkr.ecr.us-west-2.amazonaws.com/did-auth-proxy
```

Comment:
@Timtech4u Is this repository value correct? How could we check this? I would think that the repository value is correct on the `develop` branch as deployments have been working correctly in the dev environment.